doc/release/technotes/README.local.rst

```diff
@@ -15,23 +15,25 @@ communication within the construct. If communication occurs, an error
 is reported. The checks are performed in the code within the lexical
 scope of the construct, as well as in all function calls performed by
 that code, directly or indirectly, explicitly or implicitly. The
-checks can be disabled with the --no-local-checks flag, which is implied
-by the --no-checks and --fast flags.
+checks can be disabled with the ``--no-local-checks`` flag, which is implied
+by the ``--no-checks`` and ``--fast`` flags.
 
 Communication occurs in the following cases:
+
 * remote memory (i.e. data not located on the current locale)
   is referenced (read from or assigned to), or
-* an 'on' statement attempts to execute on a remote locale.
 
-The 'local' construct is useful to establish that certain code is
+* an ``on`` statement attempts to execute on a remote locale.
+
+The ``local`` construct is useful to establish that certain code is
 communication free. This may be desired, for example, when tuning
 the performance of a program, as communication usually slows down
 execution.
 
 The 'local' construct does not necessarily indicate the cause of
-communication when present. See
+communication when present. See `a link`_.
```

Comment:
I'd give this a slightly better name.  You might also be able to condense the sentence so it looks better (i.e. "See `a link`_ for ways to diagnose communication")